API.md

```diff
@@ -96,15 +109,23 @@ Takes a screenshot of the browser window where the specified web page is opened.
 Object that contains information about the browser installed on the machine.
 
 **Kind**: global typedef  
+**Note**: If BrowserInfo#winOpenCmdTemplate is defined on Windows, it will be rendered and used as open command.
```

Comment:
What about `macOpenCmdTemplate`? 
Btw, is this correct notation `BrowserInfo#winOpenCmdTemplate`? it's shown like a plain-text and doesn't looks good